crates/ty_python_semantic/src/types/class.rs

```diff
@@ -3449,6 +3465,12 @@ impl KnownClass {
             "_NoDefaultType" => Self::NoDefaultType,
             "SupportsIndex" => Self::SupportsIndex,
             "Enum" => Self::Enum,
+            "EnumMeta" if Program::get(db).python_version(db) <= PythonVersion::PY310 => {
+                Self::EnumType
+            }
```

Comment:
it's still aliased as `EnumMeta` on later versions, too, for backwards compatibility -- this is Python 3.13:

```pycon
>>> import enum
>>> enum.EnumMeta
<class 'enum.EnumType'>
```

and typeshed calls it `EnumMeta` on all Python versions: https://github.com/python/typeshed/blob/ddd8434a42b83923b770450386ee12672ab3c604/stdlib/enum.pyi#L92

```suggestion
            "EnumMeta" => {
                Self::EnumType
            }
```